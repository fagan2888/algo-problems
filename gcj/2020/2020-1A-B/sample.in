6
1
4
19
30
52
60
