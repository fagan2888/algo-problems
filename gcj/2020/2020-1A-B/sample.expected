Case #1:
1 1
Case #2:
1 1
2 1
3 2
Case #3:
1 1
2 1
3 2
4 2
5 3
5 2
5 1
4 1
Case #4:
1 1
2 1
3 2
4 2
5 3
6 3
6 2
6 1
5 1
Case #5:
1 1
2 1
3 2
4 2
5 3
6 3
7 3
7 2
8 2
8 1
Case #6:
1 1
2 1
3 2
4 2
5 3
6 3
7 4
6 4
6 5
6 6
5 5
